docs/contributing.rst

```diff
@@ -96,6 +96,26 @@ systems, ``evmone.dll`` on Windows systems and ``libevmone.dylib`` on macOS.
 On macOS some of the testing scripts expect GNU coreutils to be installed.
 This can be easiest accomplished using Homebrew: ``brew install coreutils``.
 
+Search paths
+------------
+
+The dependent libraries (such as ``evmone.so`` / ``evmone.dll``) can be
+located in different locations to be found automatically.
+
+The paths that are looked up in order are:
+
+- ``$ETH_EVMONE``
+- ``$PROJECT_ROOT/deps``
```

Comment:
Are you sure it is `PROJECT_ROOT` and not cwd?

Reply:
Yeah, it's CWD.

Reply:
oh then CWD is a bad pick, it should be changed, shouldn't it? I think depending on CWD may be bad (Don't want to bring up security concerns on this area though).

Reply:
True. I think this should depend on the location of the binary, not CWD.

Reply:
I have created an issue: #11614.

Reply:
I've actually picked #11614 up, and will resume changing the text here once after that one. @cameel

Reply:
It's not certain if we're actually going to change that so for the time being please just update the doc to mention working dir instead of `$PROJECT_ROOT`, because that's what it uses now.